scripts/link-data.sh

```diff
@@ -0,0 +1,60 @@
+#! /bin/bash
+
+# This script is used to establish symlinks for training modules to use data
+# stored in a shared directory.
+
+# Set the working directory to the directory of this file
+cd "$(dirname "${BASH_SOURCE[0]}")"
+# move back up to the training modules root
+cd ..
+
+# location for the data modules
+share_base=/shared/data
+modules_base=${share_base}/training-modules
+
+
+# RNA-seq module directories
+mkdir -p RNA-seq/data/gastric-cancer
+mkdir -p RNA-seq/data/gastric-cancer/salmon_quant
+mkdir -p RNA-seq/data/NB-cell
+mkdir -p RNA-seq/QC/gastric-cancer/fastp/
+mkdir -p RNA-seq/QC/gastric-cancer/fastqc/
+
+
+
+link_locs=(
+  RNA-seq/data/gastric-cancer/gastric-cancer_metadata.tsv
+  RNA-seq/data/gastric-cancer/fastq
```

Comment:
I guess my thought here was kind of a "why not?" They can't write anything, so that shouldn't be a problem, but this lets them see the full organization.